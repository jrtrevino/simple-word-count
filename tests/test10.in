        16        140       8688 ../Assignment4/a.out
         9        132       8656 ../Assignment3/a.out
        25        272      17344 total
