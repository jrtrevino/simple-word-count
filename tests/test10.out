        16         49        494 requirements
